templates/404.html

```diff
@@ -5,15 +5,10 @@
     <div class="row col-8">
         <h1>Ubuntu documentation</h1>
         <h2>Page not found (404)</h2>
-        <p>There is no page at this URL. How would you like to proceed?</p>
+        <p>To find what you're looking for you could try:</p>
         <ul>
-            {% if '/maas/' in request_path %}
```

Comment:
We no longer need special messages for maas or core, as maas is redirected straight over to docs.maas.io, and core is a separate application in k8s - its 404s are handled by that app.

Reply:
shouldn't we have a redirect instead?

Those pages are indexed on search engines.

Reply:
It does redirect. Did you try going to http://docs.ubuntu.com/maas/nonexistentpage? Doesn't it redirect for you?

Reply:
oh sorry I missread your comment.